app/helpers/application_helper.rb

```diff
@@ -1,2 +1,12 @@
 module ApplicationHelper
+  def bootstrap_class_for_toast(type)
```

Comment:
lefedünk minden flash message típust, amit használunk? #59 -ben az új üzenetekhez `info` és `danger` lett használva, ami itt még nem szerepel

kérlek csinálj egy rebase-t a legfrissebb masterre, majd egységesítsd a flash message típusokat. mivel a `notice`-t a devise használja, érdemes lenne megtartani, de az is jó lenne, ha összesen két verziónk maradna: egy a pozitív, egy a negatív üzeneteknek. ne feledd, kétféleképpen hívhatunk flash-t, egyrészt a `flash[...]`, másrészt a `flash.now[...]` is definiálhat új típust ha nem figyelünk oda